src/index.html

```diff
@@ -5,8 +5,391 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:regular,500,700" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="menu">
```

Comment:
add an empty line between _multiline_ siblings. Check all over the page
```suggestion
      </a>
      
      <nav class="menu">
```